src/main/java/pl/javastart/task/Vehicle.java

```diff
@@ -0,0 +1,45 @@
+package pl.javastart.task;
+
+public class Vehicle {
+    private String name;
+    private int tankCapacity;
+    private double averageFuelConsumption;
+
+    public Vehicle(String name, int tankCapacity, double averageFuelConsumption) {
+        this.name = name;
+        this.tankCapacity = tankCapacity;
+        this.averageFuelConsumption = averageFuelConsumption;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public int getTankCapacity() {
+        return tankCapacity;
+    }
+
+    public void setTankCapacity(int tankCapacity) {
+        this.tankCapacity = tankCapacity;
+    }
+
+    public double getAverageFuelConsumption() {
+        return averageFuelConsumption;
+    }
+
+    public void setAverageFuelConsumption(double averageFuelConsumption) {
+        this.averageFuelConsumption = averageFuelConsumption;
+    }
+
+    double totalDistanceTravel() {
+        return (tankCapacity / averageFuelConsumption) * 100;
+    }
+
+     String getInfo() {
```

Comment:
Checkstyle zauważył w tej linii błąd. To, co znalazł możesz podejrzeć w pull requeście z zakładce "Files changed". Polecam zainstalować sobie dodatek do intellij, żeby na bieżąco informował. Instrukcję znajdziesz tutaj https://bootcamp.javastart.pl/strona/jakosc-kodu